### PR TITLE
Alter primary key

### PR DESCRIPTION
**Pull request by Rclarkeweb**
### Type of Change

- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/sql/concepts/primary-keys/primary-keys.md
Comment (SSwiniarski) [inline content/sql/concepts/primary-keys/primary-keys.md:46]:
```suggestion
If a table doesn't have a primary key, it can be altered to add one.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Rclarkeweb I did a first review and left one comment. The rephrasing is because we don't like direct address to the reader in Docs. i.e. phrasing such as "You can..."

**Comment by Rclarkeweb** [comment]
> @Rclarkeweb I did a first review and left one comment. The rephrasing is because we don't like direct address to the reader in Docs. i.e. phrasing such as "You can..."

Thanks for letting me know! I’m happy with your changes. 👍

**Comment by Rclarkeweb** [comment]
@SSwiniarski Thanks for letting me know about the language usage! I’m happy with your changes. 👍

**Review by SSwiniarski — APPROVED** [review]
@Rclarkeweb Looks good for a second review.

**Comment by Dusch4593** [comment]
Hey @Rclarkeweb I'm going to conduct review 2️⃣ but I'm going to wait until #1465 is merged since that PR also touches the same file. Hoping to merge it by EOD at the earliest, EOW at the latest. 😄

cc: @SSwiniarski

**Comment by Rclarkeweb** [comment]
@SSwiniarski Thanks!

@Dusch4593 no problem, thanks for letting me know!

**Review by Dusch4593 — APPROVED** [review]
Thank you for your patience with us, @Rclarkeweb! This looks good to merge 😄

**Comment by Dusch4593** [comment]
@Rclarkeweb I made a few slight edits in afc934406df6013ca05c762a3c3b38fad7f4ec2b to fit your edits in with edits made to this file by another contributor in a separate PR. Included was the linking of `ALTER TABLE` to the existing [Docs entry](https://www.codecademy.com/resources/docs/sql/commands/alter-table) for it.

**Comment by Dusch4593** [comment]
Thank you for contributing to Codecademy Docs, @Rclarkeweb! 🎊 🎉 🥳 

The changes in the entry should now be live: https://www.codecademy.com/resources/docs/sql/primary-keys

I noticed that, when I was settling a merge conflict, I made a few typos. 😅 I went ahead and opened [a new PR](https://github.com/Codecademy/docs/pull/1514) to fix those. (cc: @SSwiniarski) My team and I can take of this one 👍🏻 

Looking forward to your next Docs contribution! 😄

**Comment by Rclarkeweb** [comment]
@Dusch4593 Thanks for letting me know, happy to have helped! 😄

**Comment by Dusch4593** [comment]
> @Dusch4593 Thanks for letting me know, happy to have helped! 😄

No problem! One last thing, are you seeing your profile under the contributor list?

Screen Shot 2023-01-19 at 7 25 09 PM

If you can't, and you want to make it visible, follow these steps:

1. Ensure that you are using the same email address for both your Codecademy and GitHub accounts.
2. Add your GitHub username to your Codecademy profile.
3. Log out of Codecademy.
4. Log back in through your GitHub account.

**Pull request merged**
